src/productCard.css

```diff
@@ -0,0 +1,90 @@
+.product-card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 200px;
+  color: #060b35;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+  
+.product-card__img {
+  width: 160px;
+  height: 134px;
+}
+  
+.product-card__product-name {
+  font-size: 12px;
+  font-weight: 500;
+  margin: 40px 0 0;
+  line-height: 18px;
+}
+  
+.product-card__product-code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  align-self: start;
+  margin: 4px 0 0;
+}
+  
+.product-card__reviews {
+  display: flex;
+  justify-content: space-between;
+  line-height: 14px;
+  width: 100%;
+  margin: 16px 0 0;
+}
+  
+.product-card__reviews-number {
+  font-size: 10px;
+  margin: 0;
+}
+  
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  line-height: 18px;
+  font-size: 16px;
+  margin: 24px 0 0;
+}
+  
+.product-card__price-txt {
+  font-size: 12px;
+  font-weight: 400;
+  margin: 0;
```

Comment:
You can reset the default `<p>` tag margin like you did with the <body> in style.css